src/routes/Department.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import {
+  createDepartment,
+  readAllDepartment,
+  readDepartment,
+  updateDepartment,
+  deleteDepartment,
+} from "../controllers/Department";
+
+const router = express.Router();
+
+router.post("/department", createDepartment);
+router.get("/get/:departmentId", readDepartment);
+router.get("/getAll", readAllDepartment);
+router.patch("/department/:departmentId", updateDepartment);
+router.delete("/delDepartment/:departmentId", deleteDepartment);
```

Comment:
it should be /department not /delDepartment

Reply:
request address: "/department/:id"
request type: delete
will not conflict with
request address: "/department/:id"
request type: patch